runners/google-cloud-dataflow-java/worker/src/main/java/org/apache/beam/runners/dataflow/worker/fn/control/RegisterAndProcessBundleOperation.java

```diff
@@ -126,6 +128,7 @@ public RegisterAndProcessBundleOperation(
     this.registerRequest = registerRequest;
     this.ptransformIdToSideInputReader = ptransformIdToSideInputReader;
     this.ptransformIdToSideInputIdToPCollectionView = ptransformIdToSideInputIdToPCollectionView;
+    this.sdkToDfePCollectionMapping = sdkToDfePCollectionMapping;
```

Comment:
I think all these maps functions should go under a single class. then there would be just one parameter to this class for all the maps.

Add a TODO and file a JIRA at least if its too difficult to do in this PR.

If these mappings are only used in this file then I suppose its okay though

Reply:
I was thinking of this, but I didn't come up with single purpose or name for such a class.

Reply:
Created [BEAM-6834](https://issues.apache.org/jira/browse/BEAM-6834)